src/scarab.js

```diff
@@ -152,7 +178,33 @@ var MapzenScarab = (function () {
   function _onClickGitHub (event) {
     _track('click', 'github', opts.name)
   }
+  
+  // Clicking info button should lead to pop up description to open up
+  // Clicking info button again should lead to description box closing
+  // If no description provided, do not open description box
+  function _onClickInfo(event) {
+    var elem = document.querySelector(".mz-bug-description")
+    if (elem.style.display === 'none') {
+        elem.style.display = 'block'
+    } else {
+        elem.style.display = 'none'
+    }
+  }
 
+  function _buildDescription(id) {
+    var infoBox = document.createElement('div')
+    infoBox.className = "mz-bug-" + id
+    infoBox.innerHTML = opts.description 
```

Comment:
let's use `.textContent` instead of `.innerHTML`. The value set here is user-generated content and if we allow any HTML then you run the risk of allowing malicious / insecure HTML. `textContent` will be safer. We will address things like links separately and in the future...